charts/all-in-one/templates/configmap-probe.yaml

```diff
@@ -7,5 +7,5 @@ metadata:
     app.kubernetes.io/instance: {{ $.Release.Name }}
 data:
   liveness_probe.sh: |- {{ tpl (.Files.Get "scripts/common/liveness_probe.sh") . | nindent 4 }}
-  liveness_probe_miner.sh: |- {{ tpl (.Files.Get "scripts/common/liveness_probe_miner.sh") . | nindent 4 }}
+  liveness_probe_miner.sh: |- {{ tpl (.Files.Get "scripts/common/liveness_probe_validator.sh") . | nindent 4 }}
```

Comment:
is keeping the file name key being left same intended?

Reply:
Thanks for the feedback!